src/lib/tecs/TECS.cpp

```diff
@@ -346,311 +515,200 @@ void TECS::_update_throttle_setpoint()
 	}
 
 	// ramp in max throttle setting with underspeediness value
-	throttle_setpoint = _percent_undersped * _throttle_setpoint_max + (1.0f - _percent_undersped) * throttle_setpoint;
+	throttle_setpoint = _percent_undersped * param.throttle_max + (1.0f - _percent_undersped) * throttle_setpoint;
```

Comment:
@KonradRudin This is not related to your PR but I think there is a bug above. When we don't have an airspeed sensor we just use the predicted throttle. But actually we should be using the throttle value including the damping factor, not just the feedforward, right?

Reply:
Yes, it makes sense for me, to have the p control for the altitude error included.